library/Dice.php

```diff
@@ -0,0 +1,242 @@
+<?php
```

Comment:
Please don't add an external library file in `library/`. This is a legacy artifact because I couldn't find any similar Composer libraries for ASN1 and OAuth1.

But this is a proper Composer dependency and should be added through this channel as well.

Reply:
There is no composer library available for it.
https://github.com/Level-2/Dice#installation
The author vaunts that there is no "complex" dependency necessary, just including the file.

Reply:
https://packagist.org/packages/level-2/dice

To bump the Friendica hard minimum PHP version to 7, run `composer require level-2/dice`.

Otherwise, you'll need to run `composer require level-2/dice ^2|^3` on a PHP 5 install.

Reply:
... I'm sorry, I was too stupid to find it ^^

Reply:
We don't need to keep the PHP 5 compatibility anymore. `develop` can require PHP 7 from now on and the 2019.06 version can require PHP 7.